config/features.yml

```diff
@@ -24,10 +24,10 @@ features:
     actor_type: user
     description: >
       On https://www.va.gov/find-locations/ toggles the deduplication of of services from a community care provider
-  profile_show_direct_deposit:
+  profile_schema_forms:
```

Comment:
We no longer use the `profile_show_direct_deposit` flag at all. And in fact because it was deleted from the https://api.va.gov/flipper/features dashboard it doesn't even show up there anymore even though it was still in this `features.yml` file.